docs/CONTRIBUTING.md

```diff
@@ -72,19 +56,34 @@ So the name of your new processor is `CastArithmeticOperandCheck` replacing "Che
 
 Once you have the name for the new processor, you can create a class using that
 name in `src/main/java/sorald/processor`.  This new class must extend
-`SoraldAbstractProcessor` and implement the abstract methods.
+`SoraldAbstractProcessor` and implement the abstract methods, and be annotated
+with `ProcessorAnnotation`.
 
 > See the documentation for the abstract methods in
 > [SoraldAbstractProcessor](src/main/java/sorald/processor/SoraldAbstractProcessor.java),
 > and check out [the existing implementations
 > here](/src/main/java/sorald/processor) for guidance.
 
-When you have created your processor, you must also add the check class to one
-of the four categories of check classes in the static code block in
-[Checks.java](/src/main/java/sorald/sonar/Checks.java), if it is not already
-present. To find out which category to add your check to, look it up on the
-[SonarSource website](https://rules.sonarsource.com/java), the category is
-listed just below the rule title.
+As soon as you have created a skeleton for the processor, with the
+`ProcessorAnnotation` properly filled in, you should update the `Processors`
+class. You do this by running the main method of
+`sorald.annotations.ProcessorsClassGenerator`, which you can either do with
```

Comment:
> by running the main method of `sorald.annotations.ProcessorsClassGenerator`

or by running the main method of `sorald.CodeGenerator`?

Reply:
Yep, good catch :)

Reply:
Fixed